Library/Formula/synergy.rb

```diff
@@ -0,0 +1,75 @@
+class Synergy < Formula
+  homepage 'http://synergy-project.org'
```

Comment:
We don't control the Cask's style choices, for better or worse. They're a separate project that just hop onto Homebrew's tap mechanism, essentially. Homebrew uses double quotes everywhere, attempting to follow the Ruby Style Guide.